src/sentry/runner/commands/devserver.py

```diff
@@ -116,6 +117,8 @@ def devserver(reload, watchers, workers, bind):
     from honcho.manager import Manager
 
     os.environ['PYTHONUNBUFFERED'] = 'true'
+    if refresh:
+        os.environ['REFRESH'] = 'true'
```

Comment:
I'd prefer to not name this `REFRESH`. It should be namespaced and scoped. Something like, `WEBPACK_LIVERELOAD=1`

I'd also prefer the string `1` instead of `true`. `1` is a bit more common as an environment variable name. I know `PYTHONUNBUFFERED` is right above it doing contrary, but I don't control that. :)